mongodbatlas/resource_mongodbatlas_privatelink_endpoint_service_test.go

```diff
@@ -81,11 +93,18 @@ func TestAccNetworkRSPrivateLinkEndpointServiceAWS_import(t *testing.T) {
 				),
 			},
 			{
-				ResourceName:            resourceName,
-				ImportStateIdFunc:       testAccCheckMongoDBAtlasPrivateLinkEndpointServiceImportStateIDFunc(resourceName),
-				ImportState:             true,
-				ImportStateVerify:       true,
-				ImportStateVerifyIgnore: []string{"private_link_id"},
+				ExternalProviders: map[string]resource.ExternalProvider{
+					"aws": {
```

Comment:
why do we need to provide aws as external provider here and before we didn't?

Reply:
When running the tests I faced the following error:
```        
inconsistent with the current configuration:
provider registry.terraform.io/hashicorp/aws: required by this configuration but no version is selected
```
These acceptance test do make use of the aws provider to create a `aws_vpc_endpoint`, it is likely these tests had not been run in a long time.